python/tvm/relay/op/image/image.py

```diff
@@ -125,6 +143,9 @@ def resize2d(
     size: Tuple of Int or Expr
         The out size to which the image will be resized.
 
+    roi: Tuple of Float or Expr
+        The region of interest for tf_crop_and_resize.
```

Comment:
I think itd be helpful to describe the expected shape of ROI since [2*dims] isnt obvious and is a hard requirement. Just adding it to the relay parameter comments should be enough.